### DISCUSSION TITLE
GreenLight

### DISCUSSION THREAD

**Discussion by giscus** [Jiyun-Parkk/woorifisa-fe-tech-seminar#35] [General]
# woorifisa-fe-tech-seminar/webpage-and-frontend

https://jiyun-parkk.github.io/woorifisa-fe-tech-seminar/webpage-and-frontend

**Comment by Jo-ormi** [comment]
MPA와 SPA가 무엇이고 차이가 어떤지 너무 쉽게 잘 정리해주셔서 감사합니당· ₊ ˚✿𓂃 ପ₍ᐢ๑• ֊ •๑ᐢ₎ଓ ⌒☆

**Reply by devFancy** [reply]
안녕하세요 Jo-ormi 님!
MPA와 SPA의 개념과 차이점에 대해 쉽게 설명하려고 많은 노력을 했는데, 그 부분을 좋게 봐주셔서 감사합니다!👍

**Reply by Jiyun-Parkk** [reply]
감사합니다!!
